CHANGELOG.next.asciidoc

```diff
@@ -334,6 +334,7 @@ https://github.com/elastic/beats/compare/v8.8.1\...main[Check the HEAD diff]
 - Improved Azure Blob Storage input documentation. {pull}41252[41252]
 - Make ETW input GA. {pull}41389[41389]
 - Add support for Okta entity analytics provider to collect role and factor data for users. {pull}41460[41460]
+- Add support for Journald in the System module. {pull}41555[41555]
```

Comment:
I'm not sure where it needs to be placed, but if there is a difference between he number of logs collected between journald and the log input it might be good to have it documented somewhere.

Reply:
We're already collecting all the journald logs, to me the difference in the number of logs from journald and the files is a OS detail, also some OSes only use jouranld, which makes comparing impossible unless something like `rsyslog` is installed and setup to generate the log files.